### ISSUE TITLE
Looking forward to supporting version 1.0.0

### ISSUE THREAD

**Issue by JustUse** [PGBI/kong-dashboard#194]
Looking forward to supporting version 1.0.0 : )

**Comment by julienlau** [comment]
"APIs" tab should be removed.
"plugins" tab seems to need some adjustment

**Comment by julienlau** [comment]
plugins schema changed in 1.0.0. Example:

Before:
```{
  "total": 13,
  "data": [
    {
      "created_at": 1544183783000,
      "config": {
```

Now:
```{
  "next": null,
  "data": [
    {
      "created_at": 1543939774,
      "config": {
```

**Comment by JustUse** [comment]
> 插件架构在1.0.0中已更改。例：
> 
> 之前：
> 
> ```
>   "total": 13,
>   "data": [
>     {
>       "created_at": 1544183783000,
>       "config": {
> ```
> 现在：
> 
> ```
>   "next": null,
>   "data": [
>     {
>       "created_at": 1543939774,
>       "config": {
> ```
It seems that Kong version 0.13.0 has started to abandon the API
Thank you for your reply  : )

**Comment by asanwos** [comment]
Looking forward to supporting version 1.0.0,too : )

**Comment by illyaMs** [comment]
Added in https://github.com/PGBI/kong-dashboard/pull/199
So let's wait for a feedback from @PGBI

**Comment by CuizCode** [comment]
Current version doesn't support Kong v0.15 and higher ? is there a plan ？

**Comment by PGBI** [comment]
Fixed with #199
